### ISSUE TITLE
Details of Triplets

### ISSUE THREAD

**Issue by wyhsleep** [huangwudiduan/GIMP#2]
Hi, it's a nice work, but I have a question on how did you construct the triplets set? (i.e., what is anchor $CLS_{pat}$, positive $CLS_{pat}$, and negative $CLS_{pat}$ metionde in the paper).

**Comment by huangwudiduan** [comment]
Hi！When we pre-train, we train with labels. Those with consistent labels are positive, while those with inconsistent labels are negative. Triplet loss will calculate the distance of all triplets that can be constructed in a batch.
Thanks！

发自我的iPhone


------------------ Original ------------------
From: SLEEPNOW ***@***.***&gt;
Date: Wed,Oct 11,2023 4:38 PM
To: huangwudiduan/GIMP ***@***.***&gt;
Cc: Subscribed ***@***.***&gt;
Subject: Re: [huangwudiduan/GIMP] Details of Triplets (Issue #2)





 
Hi, it's a nice work, but I have a question on how did you construct the triplets set? (i.e., what is anchor $CLS_{pat}$, positive $CLS_{pat}$, and negative $CLS_{pat}$ metionde in the paper).
